app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -71,60 +39,68 @@ To be eligible for a bursary, you’ll need at least one of the following:
 
 If your course combines 2 subjects (for example, English and history), at least 50% of your course content must be made up of the bursary subject to be eligible.
 
-You’ll also need to meet the bursary scheme’s individual terms and conditions.
+#### How bursaries are paid
 
-[Find out more about your eligibility for teaching bursaries](https://www.gov.uk/government/publications/funding-initial-teacher-training-itt/funding-initial-teacher-training-itt-academic-year-2024-to-2025).
+A bursary is paid over a minimum of 10 months by your teacher training provider. They will confirm when you will get the payments. 
 
-### Scholarships
+You will not need to pay tax or National Insurance on bursary payments. 
 
-#### Eligibility
+#### How to get a bursary
+
+You do not need to apply for a bursary. When you apply for teacher training, your provider will check if you're eligible for a bursary. If you are, your provider will confirm this in writing before you start your course. 
+
+## Eligibility for scholarships
 
 To be eligible for a scholarship, you’ll need at least one of the following:
 
 * a bachelor’s degree class 2:1 (honours) or higher
 * a master’s degree
 * a PhD
 
-In exceptional circumstances, if you achieved a 2:2 in your degree you may still be awarded a scholarship if you have significant relevant experience.
+If you achieved a 2:2 in your degree, you may still be eligible. You would need to show you have enough relevant experience and subject knowledge. 
+
+As scholarships are awarded by independent organisations, they set their own assessment requirements and application deadlines.
```

Comment:
```suggestion
Scholarships are awarded by independent organisations which set their own assessment requirements.
```

The wording as is feels a bit odd - it is almost as the `they` relates to the scholarships rather than the organisations?

But reword/disagree as needed!

Also think it is probably worth moving the bit about application deadlines to the How to apply section (we know from previous UR that this is a key bit of information)?